pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -43,6 +43,11 @@ func (ds *dockerService) getSecurityOpts(seccompProfile string, separator rune)
 	return nil, nil
 }
 
+func (ds *dockerService) getSandBoxSecurityOpts(separator rune) []string {
+	klog.Warningf("sandbox security annotations are not supported on Windows")
```

Comment:
nit: given that windows do not support these security options and they are not user defined, I would probably remove the log altogether.
```suggestion
```

Reply:
Thanks for the comments, done.